testing/test_all_components.py

```diff
@@ -0,0 +1,107 @@
+"""
+This module contains code for auto-discovering Widget classes, instantiating
+them in a programatically defined view class as a new type, which is then
+instantiated and then testing behavior of the widget objects that appear
+to get instantiated upon access.
+"""
+
+from widgetastic import attributize_string
```

Comment:
I believe this import has to come from `widgetastic.utils` - this is failing travis right now.

Though I'm not yet sure why travis results aren't being reported on your PR.